Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py

```diff
@@ -0,0 +1,223 @@
+        return True
+    now = datetime.now()
+
+    if data.get('timestamp') <= date_to_timestamp(now - timedelta(hours=client.fetch_interval_hours)):
+        return True
+
+    return False
+
+
+def test_module(client) -> str:
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: OpenPhish client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+
+    _save_urls_to_instance(client)
+    return "ok"
+
+
+def remove_backslash(url: str) -> str:
+    """
+
+    Args:
+        url: a string representing a url
+
+    Returns: the string without last '/' if such exists.
+
+    """
+    url.strip()
+    if url.endswith('/'):
+        return url[:-1]
+    return url
+
+
+def url_command(client: Client, **kwargs) -> CommandResults:
+    data = get_integration_context()
+    if _is_reload_needed(client, data):
+        reload_command(client)
+        data = get_integration_context()
+
+    url_object_list = []
+    if not data:
+        raise DemistoException("Data was not saved correctly")
+
+    url_list_from_user = argToList(kwargs.get('url'))
+    markdown = "### OpenPhish Database - URL Query\n"
+    urls_in_db = data.get('list', [])
+    for url in url_list_from_user:
+        url_fixed = remove_backslash(url)
+        if url_fixed in urls_in_db:
+            dbotscore = Common.DBotScore.BAD
+            desc = 'Match found in OpenPhish database'
+            markdown += "#### Found matches for given URL " + url + "\n"
+        else:
+            dbotscore = Common.DBotScore.NONE
+            desc = ""
+            markdown += "#### No matches for URL " + url + "\n"
+
+        dbot = Common.DBotScore(url, DBotScoreType.URL, 'OpenPhish', dbotscore, desc)
+        url_object_list.append(Common.URL(url, dbot))
+
+    return CommandResults(indicators=url_object_list, readable_output=markdown)
+
+
+def reload_command(client: Client, **kwargs) -> CommandResults:
+    _save_urls_to_instance(client)
+    return CommandResults(readable_output='updated successfully')
+
+
+def status_command(client: Client, **kwargs) -> CommandResults:
+    data = get_integration_context()
+
+    md = "OpenPhish Database Status\n"
+    if data and data.get('list', None):
+        md += "Total **" + str(len(data.get('list'))) + "** URLs loaded.\n"
+        md += "Last load time **" + timestamp_to_datestring(data.get('timestamp'),
+                                                            "%a %b %d %Y %H:%M:%S (UTC)",
+                                                            is_utc=True) + "**\n"
+    else:
+        md += "Database not loaded.\n"
+
+    return CommandResults(readable_output=md)
+
+
+demisto.debug(f'Command being called is {demisto.command()}')
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    # get the service API url
+    base_url = API_PATH
```

Comment:
we are using the global var API_PATH once. I think stating the url here, or in the client will be nicer, and no need for the global var.